mne/report.py

```diff
@@ -82,7 +82,11 @@ def _scale_mpl_figure(fig, scale):
         sfactor = -((1. / scale) ** 2)
     for text in fig.findobj(mpl.text.Text):
         fs = text.get_fontsize()
-        text.set_fontsize(fs + sfactor)
+        try:
+            text.set_fontsize(fs + sfactor)
```

Comment:
Well, did you see the magic I came up with to make it work? It's a known matplotlib issue that you cannot rescale figures with one parameter.